R/build_lookup.R

```diff
@@ -211,4 +234,6 @@ build_lookup_from_fastas <- function(data_dir, species) {
   utils::write.csv(lookup, file.path(save_dir, "lookup.csv"), row.names = FALSE)
   utils::write.csv(from_tenx, file.path(save_dir, "lookup_from_tenx.csv"), row.names = FALSE)
   utils::write.csv(from_adaptive, file.path(save_dir, "lookup_from_adaptive.csv"), row.names = FALSE)
+
+  return(save_dir)
```

Comment:
dont need to do explicit returns in R unless you want to return early from a if else block or loop - but not a critical thing to fix

Reply:
I think I finally understand how this is different in R now, and removed the unneeded ones in [c33f2ac](https://github.com/seshadrilab/tcrconvertr/pull/14/commits/c33f2ac262e79a7a1dde5e5bfe550b9eefefd912)